src/containers/applications/tabs2.scss

```diff
@@ -54,27 +54,26 @@ rediv {
   display: flex;
   flex-direction: column;
   align-items: flex-end;
-  // justify-content: flex-end;
   padding: 0 32px;
-  color: #333;
+  color: var(--dark-txt);
 }
 
 .eqCont {
   font-size: 0.22em;
-  color: #888;
+  color: var(--dark-txt);
```

Comment:
leave it as #888

Reply:
cant do that (wont work in dark mode)

Reply:
it does actually

![image](https://user-images.githubusercontent.com/54899800/140049489-058202a6-ebf0-4b4b-a157-732abcce9d25.png)

Reply:
can I use --sat-txt instead

Reply:
yes of course, what ever suits you. I just pointed out that it can be made better. The idea is not forcing global variables everytime but to rather use them as much as possible.

Reply:
dn